test/common/testlib.py

```diff
@@ -1462,6 +1477,18 @@ def nondestructive(testEntity):
     return set_obj_attribute(testEntity, "_testlib__non_destructive", True, raise_text="nondestructive", base_class=MachineCase)
 
 
+def addLink(*args):
```

Comment:
Please drop the "add" from the two new decorators -- they always add something. `@link(url)` (or perhaps more specific, `@bug(url)`) and `@tag(name)` seem fine?  Please add docstrings to say what their purpose is, especially for `tag` where it's not obvious what to do with them.

Reply:
It's already changed to suggested:
```
@FMF.link(url)
@FMF.tag("tag1", "tag2")
```
with some docstrings for these two, as probably the most important now.